WebDriverAgentLib/Categories/XCUIElement+FBTap.m

```diff
@@ -23,7 +23,8 @@ - (BOOL)fb_tapWithError:(NSError **)error
   [self fb_waitUntilFrameIsStable];
   __block BOOL didSucceed;
   [FBRunLoopSpinner spinUntilCompletion:^(void(^completion)()){
-    [[XCEventGenerator sharedGenerator] tapAtPoint:self.lastSnapshot.hitPoint orientation:self.interfaceOrientation handler:^(XCSynthesizedEventRecord *record, NSError *commandError) {
+    CGPoint hitPoint = self.isHittable ? self.lastSnapshot.hitPoint : [self coordinateWithNormalizedOffset:CGVectorMake(0, 0)].screenPoint;
```

Comment:
I remember that there was an issue between iOS versions when tapping (0, 0) offset didn't triggered button ;)